### ISSUE TITLE
Getting an error for from indicnlp import loader loader.load()

### ISSUE THREAD

**Issue by Lalit11** [anoopkunchukuttan/indic_nlp_library#19]
ModuleNotFoundError: No module named 'indicnlp.script'

**Comment by anoopkunchukuttan** [comment]
Have you set up the environment as mentioned in the README file - setting PYTHONPATH and INDIC_RESOURCES_PATH variables?

**Comment by Lalit11** [comment]
Yes, actually am working on document similarity in marathi language could
you please help me about how to get similarity between the articles ?
Thank you ,
Lalit Amrutkar ,
Pune
